kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ document.title }}</a></h1>
```

Comment:
Fixed in https://github.com/SebastianZ/kuma/commit/937a37551e8b959f7fa0f6c4464ed61a353553c5.

Sebastian